e2e/tests/functional/branding.e2e.spec.js

```diff
@@ -60,4 +60,48 @@ test.describe('Branding tests', () => {
         await page2.waitForLoadState('networkidle'); //Avoids timing issues with juggler/firefox
         expect(page2.waitForURL('**/licenses**')).toBeTruthy();
     });
+    test('Verify the Link to User\'s Guide in About Modal',
+        async ({context, page, browserName, headless}) => {
```

Comment:
Normally, we don't support conditionals in tests (which is why the linter is failing on this test). 

I think in this case, we should follow our current model https://github.com/nasa/openmct/tree/master/e2e#skipping-or-executing-tests-based-on-browser-os-andos-browser-version
add a `test.skip(browserName === 'chromium' && headless === false, 'This test cannot be executed with headless chrome')` at the beginning of this test to skip the _execution_ on headless chrome altogether.

The nightly runs all tests on all browsers and combines that code coverage with the per PR coverage, so we should still be able to track coverage of this

Reply:
Hi @unlikelyzero

 Thank you for your feedback :smile: 

I guess you meant `headless === true`. I added this to the test and now it skips for headless chromium.

Reply:
Hi @taras-z,
You can add this line directly above the `test.skip()` to bypass the lint error, since this is a valid use of `test.skip()` (we mostly have these lint rules in place to prevent accidental check-ins of skipped tests)

```
// eslint-disable-next-line playwright/no-skipped-test
```

Reply:
> 

Hi @ozyx, 
I added this fix for both tests. Is it Ok?

Reply:
Looks good to me!